CJSpohn/FitLit
# Feature/dashboard it2

Is this a fix or a feature?
Feature

What is the change/fix?
Added basic DOM functionality including displays for profile, hydration data, sleep data, activity data. Added buttons to navigate between pages. Added a sample widget to each profile page containing one of the related methods. 

Where should the reviewer start?
Scripts.js

How should this be tested?
Open index.html in terminal and see that there are currently no bugs, and that a sample widget with real user data for whoever you log in as is populated in each of our four pages. Ignore styling bugs. Those will be addressed in a future iteration.

Screenshots(if appropriate)
